src/main/java/jenkinsci/plugins/influxdb/models/Target.java

```diff
@@ -181,5 +199,59 @@ public FormValidation doCheckUrl(@QueryParameter String value) {
         public FormValidation doCheckDatabase(@QueryParameter String value) {
             return FormValidation.validateRequired(value);
         }
+        
+        @POST
+        public FormValidation doVerifyConnection(@QueryParameter String url, @QueryParameter String credentialsId,
+                                                 @QueryParameter String organization, @QueryParameter String database,
+                                                 @QueryParameter String retentionPolicy, @AncestorInPath Item context) {
+            InfluxDBClient influxDB = null;
+            doCheckUrl(url);
+            doCheckDatabase(database);
+            try {
+                Jenkins.get().checkPermission(Jenkins.ADMINISTER);
+
+                List<StandardUsernamePasswordCredentials> lookupCredentials = CredentialsProvider.lookupCredentials(
+                        StandardUsernamePasswordCredentials.class,
+                        context,
+                        ACL.SYSTEM,
```

Comment:
[ACL.SYSTEM is deprecated](https://javadoc.jenkins-ci.org/hudson/security/ACL.html#SYSTEM) and `ACL.SYSTEM2` should be used instead.

Reply:
[ACL.SYSTEM](https://javadoc.jenkins-ci.org/hudson/security/ACL.html#SYSTEM) is already used in your plugin, I suggest to do the modification in another feature.